cypress/integration/page.spec.js

```diff
@@ -64,509 +64,509 @@
+  //         .and('have.text', 'Emma de Milliano');
+
+  //       page.people().eq(3)
+  //         .find('td').eq(0)
+  //         .find('a')
+  //         .should('have.attr', 'href', '#/people/jan-van-brussel-1714')
+  //         .and('have.text', 'Jan van Brussel');
+  //     });
+
+  //     it('should not have a selected person', () => {
+  //       page.mockPeople();
+  //       page.visit('/#/people');
+
+  //       cy.get('[data-cy="person"].' + SELECTED_PERSON_CLASS)
+  //         .should('not.exist');
+  //     });
+
+  //     it('should allow to select a person', () => {
+  //       page.people().eq(1)
+  //         .find('td').eq(0)
+  //         .find('a')
+  //         .click();
+
+  //       page.assertHash('#/people/emma-de-milliano-1876')
+
+  //       page.people().eq(1)
+  //         .should('have.class', SELECTED_PERSON_CLASS);
+  //     });
+
+  //     it('should have a red link to a mother', () => {
+  //       page.people().eq(0)
+  //         .find('td').eq(4)
+  //         .contains('a', 'Maria van Brussel')
+  //         .should('have.attr', 'href', '#/people/maria-van-brussel-1801')
+  //         .and('have.class', 'has-text-danger')
+  //     });
+
+  //     it('should have a link to a father', () => {
+  //       page.people().eq(4)
+  //         .find('td').eq(5)
+  //         .contains('a', 'Emile Haverbeke')
+  //         .should('have.attr', 'href', '#/people/emile-haverbeke-1877')
+  //         .and('not.have.class', 'has-text-danger')
+  //     });
+
+  //     it('should have a text name if the mother is not in the table', () => {
+  //       page.people().eq(1)
+  //         .find('td').eq(4)
+  //         .find('a')
+  //         .should('not.exist')
+  //     });
+
+  //     it('should have a text name if the father is not in the table', () => {
+  //       page.people().eq(1)
+  //         .find('td').eq(5)
+  //         .find('a')
+  //         .should('not.exist')
+  //     });
+
+  //     it('should have an empty cell if the motherName is not given', () => {
+  //       page.people().eq(20)
+  //         .find('td').eq(4)
+  //         .should('have.text', '-')
+  //     });
+
+  //     it('should have an empty cell if the fatherName is not given', () => {
+  //       page.people().eq(20)
+  //         .find('td').eq(5)
+  //         .should('have.text', '-')
+  //     });
+  //   });
+  // });
+
+  // describe('#/people/:correct-slug page', () => {
+  //   beforeEach(() => {
+  //     page.mockPeople();
+  //     page.visit('/#/people/emma-de-milliano-1876');
+  //   });
+
+  //   it('should have correct address', () => {
+  //     page.assertHash('#/people/emma-de-milliano-1876');
+  //   });
+
+  //   it('should have People nav link active', () => {
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.assetTitle('People Page');
```

Comment:
The function `assetTitle` is likely a typo and should be `assertTitle` to correctly describe its purpose of asserting the page title.